### PR TITLE
45525803: 3D U-Net (Task 4)

### PR DESCRIPTION
**Pull request by joshknowlesuq**
Hi Shakes and the rest of the COMP3710 team,

Here is my PR for the COMP3710 report, for which I have implemented Task 4. The results of the model did not end up meeting the required 0.7 DSC but there is evidence of the model beginning to correctly segment the MRI images. I have discussed further changes that could have been made to the model in the README file.

Thanks!
Josh

### CODE REVIEW

**Comment by wooboyeong** [comment]
Code
* Poor segmentation performance
* Did you normalise the images?
* Batch normalisation may also help
* Might need to train for longer

README
* Please show the input image for the example output

General Comments
* Commit messages OK
* Pull request OK

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation, poor performance -2
## Good Practice (Design/Commenting)
Good spacing and comments
Header blocks
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No feedback required
Request Description could be more informative -1
